src/main/java/xyz/vegaone/easytrackingv2/dto/UserStory.java

```diff
@@ -22,6 +22,8 @@
 
     private List<Task> tasks;
 
+    private List<Bug> bugs;
```

Comment:
The `bugs` field in `UserStory` should match the field in `UserStoryEntity`. If they don't, the mapper will not map the field. This will likely generate an infinite loop in the mapper, which will have to be mitigated.